apps/win32/wdio.conf.js

```diff
@@ -0,0 +1,199 @@
+const path = require('path');
+
+const appPath = path.resolve(path.dirname(require.resolve('@office-iss/rex-win32/rex-win32.js')), 'ReactTest.exe');
+const appArgs = 'basePath ' + path.resolve('dist') + ' plugin defaultplugin bundle index component FluentTester';
+const appDir = path.dirname(require.resolve('@office-iss/rex-win32/rex-win32.js'));
+
+const baseUrl = 'https://webdriver.io';
+
+exports.config = {
+  runner: 'local', // Where should your test be launched
+  specs: ['../fluent-tester/src/E2E/test/**/*.ts'],
+  exclude: [
+    /* 'path/to/excluded/files' */
+  ],
+
+  maxInstances: 1,
+  capabilities: [
+    {
+      maxInstances: 1, // Maximum number of total parallel running workers.
+      platformName: 'windows',
+      deviceName: 'WindowsPC',
+      app: appPath,
+      appArguments: appArgs,
+      appWorkingDir: appDir
+    }
+  ],
+
+  /*\
+   ** ===================
+   ** Test Configurations
+   ** ===================
+   ** Define all options that are relevant for the WebdriverIO instance here
+   */
+
+  logLevel: 'info', // Level of logging verbosity: trace | debug | info | warn | error | silent
+
+  // If you only want to run your tests until a specific amount of tests have failed use bail (default is 0 - don't bail, run all tests).
+  bail: 0,
+  baseUrl: baseUrl, // Shorten url command calls by setting a base URL.
+  waitforTimeout: 10000, // Default timeout for all waitForXXX commands.
+  connectionRetryTimeout: 9000, // Timeout for any WebDriver request to a driver or grid.
+  connectionRetryCount: 2, // Maximum count of request retries to the Selenium server.
+
+  port: 4723, // default appium port
+  services: ['appium'],
+  appium: {
+    logPath: './src/reports/',
```

Comment:
I'd consider changing the log path - I don't know the right place, but I don't think it's under src

Reply:
Tracking this with a task. Once I finalize the directory structure I will fix this